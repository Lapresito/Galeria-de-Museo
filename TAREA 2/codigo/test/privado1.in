crearFecha 1/1/2021
crearPieza 70 piezaprivada1 Charles Nilson
idPieza
imprimirPieza
liberarPieza
Fin