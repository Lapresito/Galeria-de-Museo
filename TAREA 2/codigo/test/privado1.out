Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Pieza creada en forma exitosa.
3> ID: 70
4> Pieza: 70
piezaprivada1-Charles Nilson
Creada en: 1/1/2021
5> Pieza liberada en forma exitosa.
6> Fin.
